label_studio/tasks/functions.py

```diff
@@ -57,25 +61,34 @@ def redis_job_for_calculation(org_id, from_scratch, migration_name='0018_manual_
     handler.setFormatter(formatter)
     logger.addHandler(handler)
 
-    projects = Project.objects.filter(organization_id=org_id).order_by('-updated_at')
-    for project in projects:
+    project_dicts = (
+        Project.objects.filter(organization_id=org_id)
+        .order_by('-updated_at')
+        .values(
+            'id',
+            'updated_at',
+            'title',
+        )
+    )
+    for project_dict in project_dicts:
         migration = AsyncMigrationStatus.objects.create(
-            project=project,
+            project_id=project_dict['id'],
             name=migration_name,
             status=AsyncMigrationStatus.STATUS_STARTED,
         )
+        project_tasks = Task.objects.filter(project_id=project_dict['id'])
         logger.debug(
-            f'Start processing stats project <{project.title}> ({project.id}) '
-            f'with task count {project.tasks.count()} and updated_at {project.updated_at}'
+            f'Start processing stats project <{project_dict["title"]}> ({project_dict["id"]}) '
+            f'with task count {project_tasks.count()} and updated_at {project_dict["updated_at"]}'
         )
 
-        task_count = project.update_tasks_counters(project.tasks.all(), from_scratch=from_scratch)
```

Comment:
I think this would previously have set `task_count` to `None`, since `project#update_tasks_counters` launched a Redis job and returned None. This is running in a task anyway, so probably fine to switch this call to a synchronous one.